internal/pkg/pct/pct.go

```diff
@@ -74,6 +74,18 @@ type DeployInfo struct {
 	PdkInfo          PDKInfo
 }
 
+type osWrapper interface {
+	Getwd() (string, error)
+}
+type osFunc struct {
+}
+
+func (osf osFunc) Getwd() (dir string, err error) {
+	return os.Getwd()
+}
+
+var wrapper osWrapper = osFunc{}
```

Comment:
A minor drawback of this pattern is the amount of objects with similar names created. We're going to have to have a think about this - maybe using letter identifiers or something along those lines.

For me `wrapper` is a bit confusing in the context of where it's called. Right here - OK, because I can see how it was constructed above. 

Perhaps:
```go
type osWrapper interface {}
type osFunc struct {}
var osUtils osWrapper = osFunc{}
```

Reply:
name changed